src/resources/about-food-labels/view.html

```diff
@@ -0,0 +1,498 @@
+<link rel="stylesheet" href="./styles.scss">
+
+<h1>
+  About food labels
+</h1>
+
+<p>
+  Select on the food label sections below to learn about labels.
+</p>
+
+<figure>
+  <img src="https://via.placeholder.com/600x400.png/ccc/ccc" alt="" />
+</figure>
+
+<p>
+  <section class="qg-accordion flex-column" aria-label="Accordion Label">
+    
+    <div class="controls-group">
+      <input
+      type="radio"
+      name="control"
+      id="collapse"
+      class="controls collapse"
+      value="collapse"
+      role="radio">
+
+      <label for="collapse" class="controls">Collapse all</label>
+    
+      <span class="controls">|</span>
+      
+      <input
+        type="radio"
+        name="control"
+        id="expand"
+        class="controls expand"
+        value="expand"
+        role="radio">
+      
+      <label for="expand" class="controls">Expand all</label>
+    </div>
+    
+    <article>
+      <input
+        id="about-food-labels-1"
+        type="checkbox"
+        name="tabs"
+        tabindex="-1"
+        aria-controls="id-panel-content-1"
+        aria-expanded="false"
+        role="checkbox">
+      <h3
+        class="acc-heading">
+        <label for="about-food-labels-1">
+          <span class="title">
+            <span class="acc-number">1</span>
+            Food name and description
+          </span>
+          <span class="arrow"><i></i></span>
+        </label>
+      </h3>
+      
+      <div
+        class="collapsing-section"
+        aria-hidden="true"
+        id="id-panel-content-1">
+        
+        <p>
+          Food labels must show the name of the food to help identify the
+          food.
+        </p>
+
+        <p>
+          The name and description of the food must reflect its true nature.
+          For example, strawberry yoghurt must contain strawberries. If the
+          yoghurt contained strawberry flavouring rather than real fruit, then
+          the name would need to indicate that it is strawberry-flavoured
+          yoghurt.
+        </p>
+
+        <figure>
+          <img src="https://via.placeholder.com/300x300.png/ccc/ccc" alt="" />
+        </figure>
+
+        <p>
+          If the name of the food does not reflect its true nature, then you
+          must include a description of the true nature on the label. For
+          example, Luke's Hot's Sauce (red chilli pepper sauce).
```

Comment:
"Hot's" should be "Hot"

Reply:
The copy has being updated on a branch many ahead of this. Other fixes to the html have been added also.

Reply:
I pulled in the latest copy for this branch now.